attributes/default.rb

```diff
@@ -3,7 +3,19 @@
 # Attributes for InfluxDB
 
 # By default, always installs the latest by specifying nil
-default['influxdb']['version'] = nil
+default['influxdb']['version'] = '0.13.0'
+default['influxdb']['install_type'] = 'package'
+
+default['influxdb']['download_urls'] = {
+  'debian' => 'https://dl.influxdata.com/influxdb/releases',
+  'rhel' => 'https://dl.influxdata.com/influxdb/releases'
+}
+
+# platform_family keyed download sha256 checksums
+default['influxdb']['checksums']['0.10.3-1']['debian'] = '106e4884d8c654f1e43b803b21acc86e6494cafc50a46bcdb4330d0a51bd7aea'
```

Comment:
I don't think we should be keeping keys down to the version.  If anything we should just set the shasum to whatever is being set above in `['influxdb']['version']`.  No point in polluting the `node` attributes when it won't be used.

In that sense we should see attributes like this:

```
default['influxdb']['shasums'] = {
  'debian' => 'akjsdhfkjhasdljf...',
  'rhel' => 'asdfjkasdljfh....',
```